tools/topology/sof/pipe-eq-fir-capture.m4

```diff
@@ -17,6 +17,9 @@ include(`eq_fir.m4')
 # Controls
 #
 
+define(DEF_EQFIR_COEF, concat(`eqfir_coef_', PIPELINE_ID))
+define(DEF_EQFIR_PRIV, concat(`eqfir_priv_', PIPELINE_ID))
+
 # Use flat response
 include(`eq_fir_coef_flat.m4')
```

Comment:
This reads always flat and it's OK for this PR. We can align all available pipelines later. I will need them for testbench tests and I can start doing it once this system gets deployed.